README.md

```diff
@@ -51,8 +51,7 @@ YouTube Video: <How to write a Genie parser for Cisco!> https://youtube.com/watc
 Once you create a new parser, don't forget to check 1, 2, and 3;
 if you only update the parser class without modifying/creating the schema, please check 2 and 3. 
```

Comment:
need to update here, it says 1,2 and 3... and also please check other location as well to adjust to this change.

Let's add 3 like:
```
- [ ] 3. make sure GitHub Actions checks passed.

Reply:
Fixed